Source/Core/VideoBackends/Software/SWRenderer.h

```diff
@@ -13,7 +13,8 @@
 class SWRenderer : public Renderer
 {
 public:
-  ~SWRenderer() override;
+  SWRenderer();
+  ~SWRenderer();
```

Comment:
This was what I pinged you on IRC about, which I understand the reasoning for now :). I've also added the specifier to the destructors for the other derived Renderer classes.